### DISCUSSION TITLE
ClrTabs: Ability to set custom text for overflow drop down button

### DISCUSSION THREAD

**Discussion by tarangkhandelwal** [vmware/clarity#5576] [Ideas]
## Summary

The clarity ClrTabs component supports overflow capability where-in some tabs can be put behind a drop-down. This is a useful pattern in the case of a large number of tabs and limited available width. The current implementation however doesn't show the label of the selected item from within the drop down.

- What is the change?
To be able to set custom text and potentially show the selected item text upfront. This will show the selected item without clicking the drop-down and save an additional click.

- Why should it go in Clarity?
This is an enhancement for existing component behavior that Clarity supports.

- Does this change impact existing behaviors? If so how?
Should not impact existing behavior as new behavior can be input-driven

- If this change introduces a new behavior, is this behavior accessible?
Updating the label should not introduce any new accessibly issues

## Use case
We have a use case in a global search feature of our product.
We are having Nested Tabs where level 1 tabs are meant for Locations (Max 32). 
For each location, we want to show one tab for each type of Resource available in the location. (~1-100)

## Examples

_Are there any examples in other projects that demonstrate this feature or enhancement?_

## Workarounds

The component today has static clr-icon used as content of the overflow button. 
There seems to be no workaround for this.

**Comment by hippee-lee** [comment]
I converted the original issue into a discussion so we can take it into consideration as we develop Clarity Core.

**Reply by tarangkhandelwal** [reply]
I hope the use case is clear and it makes sense to use an overflow tabs pattern when there are a large number of tabs to be shown. 
The static 3 dots label doesn't help much and we should have some enhancements to support more use cases around this.

**Comment by gnomeontherun** [comment]
At first glance, yes this would be something Clarity Core would possibly support. We won't make this change in Clarity Angular at this stage, but as we consider building the Core tabs we will look at keeping this configurable.
